pkg/model/ipld.go

```diff
@@ -93,3 +94,16 @@ type IPLDMap[K comparable, V any] struct {
 	Keys   []K
 	Values map[K]V
 }
+
+func (m IPLDMap[K, V]) ToStringSlice() []string {
+	var result []string
+	for _, key := range m.Keys {
+		value, exists := m.Values[key]
+		if exists {
+			result = append(result, fmt.Sprintf("%v", key), fmt.Sprintf("%v", value))
+		} else {
+			panic(fmt.Sprintf("invalid IPLD map: key '%v' missing value", key))
```

Comment:
alternatively this could return an error, panicking here as I assume it's not possible to construct an IPLD map containing a key without a value, I am probably wrong on this.

Reply:
Even if you create an IPLDMap with a nillable V then you should still be able to retrieve it and sprintf it as %v - seems unlikely to have a key with no value.  But might be best to return an error, panicking is probably a bit excessive :D